SwiftWisdomTests/BitMaskTests.swift

```diff
@@ -34,9 +34,9 @@ class BitMaskTests: XCTestCase {
     }
     
     func testDataInitializer() {
-        let data = "0x1f".ip_dataFromHexadecimalString()!
-        let mask = UInt8(ip_data: data)
-        let hex = data.ip_hexInt!
+        let data = "1f".ip_dataFromHexadecimalString()!
+        let mask = UInt8(ip_data: data)!
```

Comment:
I've always thought force unwrapping was ok in tests, because any nil value unwrapped would act like a failure of the tests.